packages/next/package.json

```diff
@@ -78,7 +78,9 @@
     "caniuse-lite": "^1.0.30001228",
     "chalk": "2.4.2",
     "chokidar": "3.5.1",
+    "cli-select": "1.1.2",
```

Comment:
This should be a `devDependency` since it's ncc'd right?

Reply:
Ah good point, fixed